torchci/lib/bot/autoLabelBot.ts

```diff
@@ -375,16 +377,12 @@ function myBot(app: Probot): void {
 
   app.on("pull_request_review.submitted", async (context) => {
     // Apply `ciflow/trunk` to PRs in PyTorch/PyTorch that has been reviewed
-    if (context.payload.review.state !== "approved") {
-      return;
-    }
```

Comment:
Why are you removing this one? It will apply the label even if changes are requested, wouldn't it?